### DISCUSSION TITLE
How to download folder

### DISCUSSION THREAD

**Discussion by rootsmusic** [community/community#24067] [Apps, API and Webhooks]
<p>How can I download a folder from a repo that I don’t own?  Thanks.</p>

**Comment by gitstua** [comment, marked as answer]
<p>This is a tool I have used before <a href="https://downgit.github.io" rel="noopener nofollow ugc">https://downgit.github.io</a> - it also allows you to create a link for others to download a folder as a ZIP file</p>
<p>There are a number of others mentioned here: <a href="https://stackoverflow.com/a/18194523" class="inline-onebox" rel="noopener nofollow ugc">git - Download a single folder or directory from a GitHub repo - Stack Overflow</a></p>
